proxy-node-gateway/src/main/java/uk/gov/ida/notification/GatewayApplication.java

```diff
@@ -175,6 +174,9 @@ private void registerResources(GatewayConfiguration configuration, Environment e
 
         SamlRequestSignatureValidator samlRequestSignatureValidator = createSamlRequestSignatureValidator(connectorMetadataResolverBundle);
 
+        Client translatorClient = new JerseyClientBuilder(environment).using(configuration.getTranslatorServiceConfiguration().getClient()).build("translator");
```

Comment:
This is a very long line, `configuration.getTranslatorServiceConfiguration()` is shared between this and the next statement maybe bring it out to a variable first?

Reply:
My next commit should sort this out in a better way.